fuel/converters/youtube_audio.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+import h5py
+import scipy.io.wavfile
+
+from fuel.converters.base import fill_hdf5_file
+
+
+def convert_youtube_audio(directory, output_directory, youtube_id, channels,
+                          sample, output_filename=None):
+    input_file = os.path.join(directory, '{}.m4a'.format(youtube_id))
+    wav_filename = '{}.wav'.format(youtube_id)
+    wav_file = os.path.join(directory, wav_filename)
+    command = "ffmpeg -y -i {} -ac {} -ar {} {}".format(
```

Comment:
Yeah, I briefly looked at pyffmpeg and it came with a warning saying "will probably not compile with any recent version of ffmpeg" which didn't sound promising.

I can add a check to see if ffmpeg is available and whether the input file exists and give sensible errors if either is false, but otherwise I don't see an easy way around calling this command in a subshell.